build.sbt

```diff
@@ -2,11 +2,27 @@ lazy val scala212 = "2.12.18"
 ThisBuild / crossScalaVersions := Seq(scala212)
 ThisBuild / scalaVersion := scala212
 ThisBuild / dynverSonatypeSnapshots := true
-ThisBuild / version := {
-  val orig = (ThisBuild / version).value
-  if (orig.endsWith("-SNAPSHOT")) "0.10.0-SNAPSHOT"
```

Comment:
This is useful for local development to keep the version number stable.

Reply:
I was going to comment on what this is for, but aren't we getting the version from git tags? i.e. if I run 

```
<@sbt-osgi>-<⎇ use-sbt-github-actions>-> sbt version
[info] welcome to sbt 1.9.6 (Homebrew Java 11.0.20.1)
[info] loading global plugins from /Users/mdedetrich/.sbt/1.0/plugins
[info] loading settings for project sbt-osgi-build from plugins.sbt ...
[info] loading project definition from /Users/mdedetrich/github/sbt-osgi/project
[info] compiling 1 Scala source to /Users/mdedetrich/github/sbt-osgi/project/target/scala-2.12/sbt-1.0/classes ...
[info] loading settings for project sbt-osgi from build.sbt ...
[info] set current project to sbt-osgi (in build file:/Users/mdedetrich/github/sbt-osgi/)
[info] 0.9.6+8-60bd5eda-SNAPSHOT
```

Which is what is intended?

Reply:
For sbt plugins, I prefer having the same snapshot version, or else publishLocal would keep creating a new one during `scripted` test. Dynver is still in effect for non-snapshots.

Reply:
Oh I see what you mean, I think there is a more idiomatic way to do this with dynver. Let me look into it.

Reply:
@eed3si9n I have solved this in a more principled way (see https://github.com/sbt/sbt-dynver#custom-version-string). Let me know if its acceptable.